src/components/UserSelector/UserSelector.tsx

```diff
@@ -0,0 +1,82 @@
+import classnames from 'classnames';
+import React, { useContext } from 'react';
+import { TodosContext } from '../../TodoContext/TodoContext';
+import { User } from '../../types/User';
+
+export const UserSelector: React.FC = () => {
+  const {
+    users,
+    setSelectedUser,
+    selectedUser,
+    isDropdownActive,
+    setIsDropdownActive,
```

Comment:
you use it only here - in the `UserSelector` component

So move this state from context to the component

Check all state values in context - maybe there are more similar issues